jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py

```diff
@@ -86,8 +89,10 @@ def use_table(self):
             try:
                 cursor.execute(
                     f'CREATE TABLE {self.table} ( \
-                    ID VARCHAR PRIMARY KEY,  \
-                    DOC BYTEA);'
+                    ID VARCHAR PRIMARY KEY  \
+                    , EMBEDDING BYTEA  \
+                    , DOC BYTEA  \
+                    );'
```

Comment:
This is a breaking change. Could we also release a tag-specific one without changing the table columns?  As, our EAP custom already has millions of docs stored n PostgresStorage.

Reply:
I just got some feedbacks from our eap partner, they said they can re-index their data. So I think we are free to release the new psql indexer